### PR TITLE
Update README.md with Digital Ocean example

### LINKED ISSUE #610: Support for DigitalOcean Container Registry
### Description

Would be great if the [DigitalOcean Container Registry](https://www.digitalocean.com/products/container-registry) was supported officially by having a dedicated usage section in the README!

### PR DESCRIPTION
**Pull request by JohnGUnderwood**
Resolves #610 

Tested in the following action setup:
```
name: Build and Push

on:
  release:
    types: [published]

jobs:
  build_and_push:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout the repo 
        uses: actions/checkout@v4

      - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v3

      - name: Extract metadata (tags, labels) for Docker
        id: meta
        uses: docker/metadata-action@v5
        with:
          images: registry.digitalocean.com/junderwood/website
          
      - name: Login to DigitalOcean Container Registry
        uses: docker/login-action@v3
        with:
          registry: registry.digitalocean.com
          username: ${{ secrets.DIGITALOCEAN_USERNAME }}
          password: ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}
          
      - name: Build and push
        uses: docker/build-push-action@v5
        with:
          context: .
          platforms: linux/amd64,linux/arm64
          file: ./Dockerfile
          push: true
          tags: ${{ steps.meta.outputs.tags }}
          labels: ${{ steps.meta.outputs.labels }}
 ```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -471,6 +472,28 @@ jobs:
           password: ${{ secrets.QUAY_ROBOT_TOKEN }}
 ```
 
+### DigitalOcean Container Registry
+Use your DigitalOcean registered email address and an API access token to authenticate.
```
Comment (crazy-max) [inline README.md:476]: missing new line before and after this one

**Review by crazy-max — CHANGES_REQUESTED** [review]
small nit otherwise LGTM, thanks!

File: README.md (hunk 1 above)
Comment (JohnGUnderwood) [reply README.md:477]: Fixed the formatting issue!

**Review by crazy-max — CHANGES_REQUESTED** [review]
Thanks, can you squash your commits please? Otherwise LGTM

**Comment by JohnGUnderwood** [comment]
I believe I merged those two commits correctly.

**Review by crazy-max — APPROVED** [review]
LGTM thanks!

**Pull request merged**